grassroots-backend/src/mikro-orm.config.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable check-file/filename-naming-convention */
```

Comment:
This is required for the mikro orm CLI to work, for running migrations and stuff. A bit annoying that we need to read the env file in two places, but I think it's probably fine.